internal/tpl/funcs.go

```diff
@@ -311,7 +311,7 @@ func date(ts time.Time) string {
 
 func truncate(length int, v any) string {
 	sv := strval(v)
-	if len(sv) < length-3 {
+	if len(sv) < length {
```

Comment:
I suspect the intention was the avoid the risk of `length-3` below on line 318 being negative when e.g. `len(sv) == 2` but `length == 1` ? But even then that would have needed fixing, good catch.

Reply:
No, not initially. But I just ran into that as well.